src/style.css

```
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
   margin: 0;
 }
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  background-color: white;
```

```suggestion
  background-color: #fff;
```

Use hex colors everywhere to match all browsers